### ISSUE TITLE
Error: NefitEasy: Error getting http data!

### ISSUE THREAD

**Issue by Handzoff** [robertklep/nefit-easy-http-server#36]
Sorry in advance for dumb questions caused by my noobness.

I'm trying to install the Nefit Easy server for use in Domoticz with the following install guide:
[https://www.domoticz.com/wiki/NefitEasy](url)
The Easy Server starts but in the Domoticz log I keep seeing: 
> Error: NefitEasy: Error getting http data!

The Nefit Easy server is running, but when I test it using:

> curl http://127.0.0.1:3000/bridge/ecus/rrc/uiStatus
I get a can't connect error.

My Android Nefit Easy app works, so serial, access-key and password are OK. I also double-checked for typo's.

There were some install errors with the "sudo npm i nefit-easy-http-server -g" command saying that some XMPP packages were deprecated? Maybe something went wrong during install.
I tried reinstalling node version 12.x, but that did not work. Uninstalled node and reinstalled node version 10.x but that also did not work. I'm afraid I messed up dependencies with several rounds of installing and de-installing and I have no idea how to get back to clean slate before re-installing from scratch again. I also read on the Domoticz forum that PM2 is not necessary? But then how to start the nefit-easy-http-server? 

Any help is greatly appreciated!

**Comment by robertklep** [comment]
There's a difference between errors and warnings. Typically, warnings will not cause issues. Only errors will. So a deprecation warning shouldn't prevent the program from running.

Did you run the `curl` program from the same machine that's running the Easy server? Otherwise, the `127.0.0.1` address won't work and you need to use the machine's IP-address on your local network.

You can (test-)run the program without PM2 like this:
```
easy-server --serial=YOURSERIAL --access-key=YOURACCESSKEY --password=YOURPASSWORD --host=0.0.0.0
```

**Comment by Handzoff** [comment]
Well, no joy yet. I reinstalled node.js again and reinstalled nefit easy server again with `sudo npm i nefit-easy-http-server -g`

The `curl http://127.0.0.1:3000/bridge/ecus/rrc/uiStatus` command was run from my RPi 3B+ and it returns
`curl: (7) Failed to connect to 127.0.0.1 port 3000: Connection refused`

When I try to run the easy-server with the following command:
`easy-server --serial=YOURSERIAL --access-key=YOURACCESSKEY --password=YOURPASSWORD --host=0.0.0.0`
It just hangs, no cursor...and I need to press CTRL-C to get cursor back.

`pi@raspberrypi:~ $ curl http://127.0.0.1:3000/bridge/ecus/rrc/uiStatus
curl: (7) Failed to connect to 127.0.0.1 port 3000: Connection refused
pi@raspberrypi:~ $ easy-server --serial=xxxxxxxxx --access-key=xxxxxxxxxxxxxxxx --password=xxxxxx --host=127.0.0.1`

Domoticz still returns: `Error: NefitEasy: Error getting http data!`
I can reach the Bosch server: `Connection to wa2-mz36-qrmzh6.bosch.de 5222 port [tcp/xmpp-client] succeeded!
`
I'm stumped.....

**Comment by Handzoff** [comment]
Wohoo! It's working!
I followed your advice in the closed issue 35: `keeps trying to setup connection / loop #35`

As you suggested at the end of the closed issue:
`The OpenSSL defaults for Buster are causing the problem, probably because Nefit/Bosch is using either an outdated TLS version or an outdated cipher. The proposed fix for /etc/ssl/openssl.cnf fixes the issue with easy-server.` 

Changed the settings back to suggested values and Domoticz finds the Nefit Easy sensors.
Thanks for your support.
